docs/howto/acuity_ramp.md

```diff
@@ -0,0 +1,122 @@
+
+    A system deployer organization with few initial resources might start with a simple decision model that only includes a custom
+    `IN_KEV` decision point. The `IN_KEV` decision point would be a simple binary indicator of whether a vulnerability
+    has been added to CISA's [Known Exploited Vulnerabilities](https://www.cisa.gov/known-exploited-vulnerabilities-catalog)
+    ([KEV](https://www.cisa.gov/known-exploited-vulnerabilities-catalog)) catalog. Because this information is free,
+    publicly available, and because it is a simple binary indicator, it is easy to gather and analyze even for a very
+    small organization.
+
+    The following table shows how the organization might expand their decision model as they grow in capability.
+    
+    | Acutiy Level | Tree |
+    | --- | --- |
+    | 1 | ```[IN_KEV]``` |
+    | 2 | ```[EXPLOITATION_1]``` |
+    | 3 | ```[EXPLOITATION_1, SYSTEM_EXPOSURE_1_0_1]``` |
+    | 4 | ```[EXPLOITATION_1, SYSTEM_EXPOSURE_1_0_1, AUTOMATABLE_2]``` |
+    | 5 | ```[EXPLOITATION_1, SYSTEM_EXPOSURE_1_0_1, AUTOMATABLE_2, MISSION_IMPACT_2, SAFETY_IMPACT_1]```
+
+    !!! tip "Acuity Levels are Stakeholder-Specific" 
+
+        This example is demonstrating the _concept_ of _acuity levels_ in SSVC adoption. We are specifically
+        _not_ saying that there are 5 levels of acuity in SSVC; in principle this concept could be applied to any number
+        of levels (including just one). In practice, the number of levels necessary would be a stakeholder-specific implementation choice, so 
+        there is no inherent meaning to the "Acuity Levels" shown here outside the context of this example.
+    
+    The remainder of this example shows one path the organization might take to grow their decision model
+    according to the table above.
+
+    ### Improved Exploit Awareness (Acuity Level 2)
+
+    As the organization becomes more capable of gathering and analyzing data, they might start collecting their own
+    data on exploitation, allowing them to move to a more complex decision tree that replaces the binary `IN_KEV` 
+    decision point with the trinary `EXPLOITATION_1` decision point.
+
+    {% include-markdown "../_generated/decision_points/exploitation_1_0_0.md" %}
+
+    ### Improved Asset Management (Acuity Level 3)
+
+    As they continue to develop their internal asset management capabilities, they might find they have enough
+    asset data to reflect the degree to which a system is exposed to the internet, allowing them to 
+    incorporate the `SYSTEM_EXPOSURE_1_0_1` decision point into their decision model.
+
+    {% include-markdown "../_generated/decision_points/system_exposure_1_0_1.md" %}
+
+    ### Improved Threat and Vulnerability Analysis (Acuity Level 4)
+
+    Over time, the organization's threat and vulnerability analysis capabilities might reach a point where they can
+    begin to collect data on the degree to which a vulnerability is automatable, allowing them to incorporate the
```

Comment:
Should we also add a comment that they might not have to analyze this information themselves, but they would have to be able to integrate external vulnerability data sources such as the NVD and possibly write a script that translates CVSS v3 or v4 metric strings into a value for automatable?

Reply:
see 80efd8b